cdap-watchdog/src/main/java/io/cdap/cdap/logging/logbuffer/LogBufferService.java

```diff
@@ -99,6 +99,24 @@ public class LogBufferService extends AbstractIdleService {
 
   @Override
   protected void startUp() throws Exception {
+    initializeAndStartServices(
+        this.handlers,
+        cConf.get(Constants.LogBuffer.LOG_BUFFER_SERVER_BIND_ADDRESS),
+        cConf.getInt(Constants.LogBuffer.LOG_BUFFER_SERVER_BIND_PORT)
+    );
+  }
+
+  /**
+   * Initializes and starts the log buffer related services, including log pipelines, recovery
+   * service, concurrent writer, and HTTP service.
+   *
+   * @param additionalHandlers Additional HTTP handlers to be registered.
+   * @param address            The address to bind the HTTP service to.
+   * @param port               The port to bind the HTTP service to.
+   * @throws Exception If an error occurs during initialization or startup.
+   */
+  public void initializeAndStartServices(Set<HttpHandler> additionalHandlers, String address,
```

Comment:
No need for this wrapper `initializeAndStartServices`. Callers should call `start()` or `startAndWait()` for the LogBuferService.